bundle/jsky.app.ot/src/main/scala/jsky/app/ot/ags/BagsManager.scala

```diff
@@ -171,29 +193,39 @@ final class BagsManager(executorService: ExecutorService) {
                       if (!state.keys(key)) {
                         LOG.info(s"$bagsIdMsg successful. Results=${opt ? "Yes" | "No"}.")
                         applySwingResults(opt)
+                        clearStatus()
                       }
 
+                    // Note that we have to sleep after reporting a bags failure, as otherwise, there is no time for
+                    // the failure status message to be read before we change back to pending.
+
                     // We don't want to print the stack trace if the host is simply unreachable.
                     // This is reported only as a GenericError in a CatalogException, unfortunately.
                     case Failure(CatalogException((e: GenericError) :: _)) =>
                       LOG.warning(s"$bagsIdMsg failed: ${e.msg}")
+                      status(BagsStatus.Failed("Catalog lookup failed."))
+                      Thread.sleep(3000)
```

Comment:
Instead of sleeping perhaps the pending and running status could include the results of the last attempt.  Like  "Catalog lookup failed. Will retry."  "Catalog lookup failed.  Retrying ..." or something.